Runtime/JSException.cs

```diff
@@ -14,4 +18,43 @@ public unsafe JSException(string message)
     {
         ErrorInfo = new JSErrorInfo(message, JSStatus.GenericFailure);
     }
+
+    [DoesNotReturn]
+    public static void Fatal(string message, string location)
+        => napi_fatal_error(location, NAPI_AUTO_LENGTH, message, NAPI_AUTO_LENGTH);
```

Comment:
If two parameters are specified, then will the call be ambiguous due to the default values in the overload below?

Reply:
Oops... It is still work in progress. I will see if I can remove all this "drafty"

Reply:
I removed this overload.